src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/Utgiftsperiode/UtgiftsperiodeValg.tsx

```diff
@@ -48,6 +63,21 @@ const UtgiftsperiodeValg: React.FC<Props> = () => {
                 <Label>Velg barn</Label>
                 <Label>Ant.</Label>
                 <Label>Utgifter</Label>
+                {utgiftsperioderState.value.map((utgiftsperiode, index) => {
+                    const { periodetype } = utgiftsperiode;
+                    return (
+                        <React.Fragment key={index}>
+                            <PeriodetypeSelect
+                                className={'ny-rad'}
```

Comment:
hva brukes denne til?

Reply:
For å bestemme at øverste element skal starte i kolonne 1 på neste rad dersom det ikke er nok elementer til å fylle opp en rad i grid. Den ligger i `Grid` stylingen som skal tas senere, så mulig den forsvinner.